packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
 ## Unreleased
+* Changed
+  * Removed ability to set variable row heights
 
 * Initial stable release
```

Comment:
Since this is the first PR after release, can we "fix" a mistake in the CHANGELOG?
```suggestion
## Unreleased
* Changed
  * Removed ability to set variable row heights

## 0.1
  * Initial stable release
```